lib/k8s/stack.rb

```diff
@@ -67,8 +71,21 @@ def prepare_resource(resource, base_resource: nil)
       })
     end
 
+    # @param client [K8s::Client] apply using client
+    # @param prune [Boolean] prune automatically after apply
     # @return [Array<K8s::Resource>]
     def apply(client, prune: true)
+      specials, others = resources.partition { |r| SPECIAL_KINDS.include?(r.kind) }
```

Comment:
"Resource Definitions" would be a better term than "special" - these are resources that **define** other resource kinds.

Reply:
Fixed.